### DISCUSSION TITLE
Commission on a test payment gateway!?!?!

### DISCUSSION THREAD

**Discussion by offlink-io** [swellstores/community#507] [Help] [answered]
Hello,

I have just received an invoice indicating an amount to be paid as I have made sales amounting to $1,508,554:

![image](https://github.com/swellstores/community/assets/128164601/9196d7dd-4c3b-4739-bffa-95cab177f63c)

However, **these are tests carried out on a test gateway!**!!!

![image](https://github.com/swellstores/community/assets/128164601/a80df121-204f-46bd-bf7c-b8b592a35315)

It's a good thing I didn't test for larger amounts!

Please let me know how to get my money back.

I can't contact you directly on support because my plan doesn't allow it.

I am very surprised by this ...

My store: la-rue-vers-le-bois

**Comment by ericingram** [comment]
👎 

Shouldn't happen on a test gateway, but there are some old configurations where we believe it could be possible. I forwarded this to our support team so don't worry, we'll get it sorted out for you.

**Reply by logeshswell** [reply]
Test gateway won't apply to manual payment methods and the orders paid through manual payment methods will be accounted as sales revenue.

order_id - grand_total (€) - grand_total ($)
* 65c6554eb569d0001270641d - 231.19 - 249.3287
* 65bec720a433f40012645fd6 - 47.99 - 51.8615
* 65bdf917d105dd00128b623c - 182.35 - 197.0606
* 65bd6046a141fe0011330d24 - 119.97 - 130.4603
* 65b91c12d2c4150011bc0501 - 154 - 166.8418
* 65acebe61fc2070012c8ec46 - 704.5 - 764.863

The above (**non-test**) orders are accounted for costing $1560.41 and all were paid through manual payment methods like cash, bank_cheque, account and cash_on_delivery.

The below 2 orders were deleted from your store after it is paid, but still, they would be accounted.
* 65b91c12d2c4150011bc0501
* 65acebe61fc2070012c8ec46

**Reply by offlink-io** [reply]
Hello and thank you for your reply.

Any way to test manual payment methods without them being considered as real payments?

I must confess that at the moment I'm just testing the implementation of my solution on yours.

Thx

**Reply by ericingram** [reply, marked as answer]
If you're creating manual payments using the API (`/payments` endpoint), you can pass `test: true`

**Reply by offlink-io** [reply]
Ok thx for your response

**Reply by ericingram** [reply]
One more point: if the order is canceled instead of deleted, then the system will subtract it from sales calculations.
